website/layouts/_partials/sidebar/toc-data.html

```
@@ -0,0 +1,55 @@
+      {{- end -}}
+    {{- else if lt $d $prevDepth -}}
+      {{- range seq 1 (sub $prevDepth $d) -}}
+        {{- $html = printf "%s</li></ul>" $html -}}
+      {{- end -}}
+      {{- $html = printf "%s</li>" $html -}}
+    {{- else -}}
+      {{- $html = printf "%s</li>" $html -}}
+    {{- end -}}
+
+    {{- if .label -}}
+      {{- $html = printf "%s<li><strong>%s</strong>" $html .label -}}
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </li> ], start tag match failed [ <li> ] on line 39.

(tag-pair)